addons/hr/models/hr_employee.py

```diff
@@ -100,7 +100,7 @@ def _lang_get(self):
         ('present', 'Present'),
         ('absent', 'Absent'),
         ('archive', 'Archived'),
-        ('out_of_working_hour', 'Off-Hours')], compute='_compute_presence_state', default='out_of_working_hour')
+        ('out_of_working_hour', 'Off-Hours')], store=True, compute='_compute_presence_state', default='out_of_working_hour')
```

Comment:
The field becomes stored, but it depends on time, so if nothing is triggering the recompute, the presence state is gonna be permanent (eg: Present if not there etc).